<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Начнём!</string>
  </property>
  <widget class="QPushButton" name="new_game_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Новая игра</string>
   </property>
  </widget>
  <widget class="QLabel" name="greeting_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="instruction_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Инструкция</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_game_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить игру</string>
   </property>
  </widget>
  <widget class="QPushButton" name="statistics_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотр статистики</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
